src/Ansatz/Chain.jl

```diff
@@ -198,22 +198,14 @@ for i < center the tensors are left-canonical and for i > center the tensors are
 and in the center there is a matrix with singular values.
 """
 function mixed_canonize!(::Open, tn::Chain, center::Site) # TODO: center could be a range of sites
-    N = length(sites(tn))
-
-    # Left-to-right QR sweep -> get left-canonical tensors
-    for i in 1:N-1
-        canonize_site!(tn, Site(i); direction = :left, mode = :qr)
+    # left-to-right QR sweep (left-canonical tensors)
+    for i in 1:center.id-1
+        canonize_site!(tn, Site(i); direction = :right, method = :qr)
```

Comment:
This is the other way around:
```suggestion
        canonize_site!(tn, Site(i); direction = :left, method = :qr)
```